client/src/unifycr.c

```diff
@@ -1578,67 +1599,67 @@ static void* unifycr_superblock_bgq(size_t size, const char* name)
     unifycr_init_pointers(shmptr);
 
     /* initialize data structures within block if we haven't already */
-    uint32_t* header = (uint32_t*) shmptr;
+    uint32_t *header = (uint32_t *) shmptr;
     uint32_t magic = *header;
     if (magic != 0xdeadbeef) {
-      unifycr_init_structures();
-      *header = 0xdeadbeef;
+        unifycr_init_structures();
+        *header = 0xdeadbeef;
     }
 
     return shmptr;
 }
 #endif /* MACHINE_BGQ */
 
 /* converts string like 10mb to unsigned long long integer value of 10*1024*1024 */
-static int unifycr_abtoull(char* str, unsigned long long* val)
+static int unifycr_abtoull(char *str, unsigned long long *val)
 {
     /* check that we have a string */
     if (str == NULL) {
         debug("scr_abtoull: Can't convert NULL string to bytes @ %s:%d",
-            __FILE__, __LINE__
-        );
+              __FILE__, __LINE__
+             );
```

Comment:
Don't have to do it in this PR, but it would be nice to go back and move up all these closing parentheses on their own lines at some point.